CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.14.0
 
+- Adds new 'limiter' class for setting max page width across screen sizes.
+- Significantly refactors layout scale to be more consistent across different properties and to increment more predictibly.
```

Comment:
I didn't want to do this because the documentation covers the specific changes! But if you think it'll be helpful, I will!